From: Vincent Tremblay <vincent@vtremblay.dev>
To: unlisted-recipients:; (no To-header on input)
Cc: vincent@vtremblay.dev, "Rob Herring" <robh+dt@kernel.org>,
	"Krzysztof Kozlowski" <krzysztof.kozlowski+dt@linaro.org>,
	"Mark Brown" <broonie@kernel.org>,
	"Guenter Roeck" <linux@roeck-us.net>,
	"Stanislav Jakubek" <stano.jakubek@gmail.com>,
	"Marcello Sylvester Bauer" <sylv@sylv.io>,
	"Marek Vasut" <marex@denx.de>,
	"Samuel Holland" <samuel@sholland.org>,
	"Conor Dooley" <conor.dooley@microchip.com>,
	"Pali Rohár" <pali@kernel.org>,
	devicetree@vger.kernel.org, linux-kernel@vger.kernel.org,
	linux-spi@vger.kernel.org
Subject: [PATCH 2/2] spidev: Add Silicon Labs SI3210 device compatible
Date: Tue, 27 Dec 2022 09:10:08 -0500	[thread overview]
Message-ID: <20221227141011.111410-2-vincent@vtremblay.dev> (raw)
In-Reply-To: <20221227141011.111410-1-vincent@vtremblay.dev>

Add compatible string for Silicon Labs SI3210 device.

Signed-off-by: Vincent Tremblay <vincent@vtremblay.dev>
---
 drivers/spi/spidev.c | 2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -701,6 +701,7 @@ static const struct spi_device_id spidev
 	{ .name = "spi-petra" },
 	{ .name = "spi-authenta" },
 	{ .name = "em3581" },
+	{ .name = "si3210" },
 	{},
 };
 MODULE_DEVICE_TABLE(spi, spidev_spi_ids);
@@ -728,6 +729,7 @@ static const struct of_device_id spidev_
 	{ .compatible = "cisco,spi-petra", .data = &spidev_of_check },
 	{ .compatible = "micron,spi-authenta", .data = &spidev_of_check },
 	{ .compatible = "silabs,em3581", .data = &spidev_of_check },
+	{ .compatible = "silabs,si3210", .data = &spidev_of_check },
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
